web/router.go

```diff
@@ -346,6 +347,15 @@ func Router() *echo.Echo {
 	e := echo.New()
 	e.HideBanner = true
 	e.HidePort = true
+	topo := viper.GetString("access_topology")
+	switch topo {
+	case "direct":
+		e.IPExtractor = echo.ExtractIPDirect()
+	case "xff":
+		e.IPExtractor = echo.ExtractIPFromXFFHeader()
+	case "xrip":
+		e.IPExtractor = echo.ExtractIPFromRealIPHeader()
+	}
```

Comment:
I wonder if we should add a default here that print an error message warning about legacy behaviour fallback (which is not a good choice according to [documentation](https://echo.labstack.com/guide/ip-address/).

This should not happen when `access_topology` is not defined because of default value being `direct` but could occur if set to whatever string outside of the 3 known values.